### PR TITLE
Return full size of the matched area

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
I think this might be more handy for the clients

### CODE REVIEW

File: lib/image-util.js [hunk 1]
```diff
@@ -360,7 +360,8 @@ async function getImagesSimilarity (img1Data, img2Data, options = {}) {
 
 /**
  * @typedef {Object} OccurrenceResult
- * @property {Point} location The absolute coordinates of the partial image occurence
+ * @property {Rect} location The absolute location of the partial image occurence
```
Comment (imurchie) [inline lib/image-util.js]: This seems like a confusing name, now, since it is not a "location" if it includes a width and height.

Comment (mykola-mokhnach) [reply lib/image-util.js]: which one would be better?

Comment (imurchie) [reply lib/image-util.js]: Well, throughout we (and Selenium) have used `rect` as the thing which has a position and dimensions.

Comment (mykola-mokhnach) [reply lib/image-util.js]: ok, changed to `rect`

**Comment by imurchie** [comment]
Published in `appium-support@2.15.0`.

**Pull request merged**